client/src/js/users/components/Item.js

```diff
@@ -1,19 +1,24 @@
 import React from "react";
 import { connect } from "react-redux";
 import { LinkContainer } from "react-router-bootstrap";
-import { ListGroupItem } from "react-bootstrap";
 import { get } from "lodash-es";
-import { Flex, FlexItem, Identicon, Icon } from "../../base";
+import styled from "styled-components";
+import { Flex, FlexItem, Identicon, Icon, LinkBox } from "../../base";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['Flex' is defined but never used.](https://app.codacy.com/manual/virtool/virtool/pullRequest?prid=4903821)